packages/cosmic-swingset/src/check-lmdb.js

```diff
@@ -51,8 +51,8 @@ export function getBestSwingStore(tempdir) {
     console.log(`LMDB does not work, falling back to Simple DB`, e);
     // see https://github.com/Agoric/agoric-sdk/issues/950 for details
     return {
-      openSwingStore: openSwingStoreSimple,
-      initSwingStore: initSwingStoreSimple,
+      openSwingStore: () => openSwingStoreSimple(),
+      initSwingStore: () => initSwingStoreSimple(),
```

Comment:
we might want to check with @michaelfig and delete this fallback, if the simpleStore won't be persistent (or can persist kvStore but not transcripts)

Reply:
Are we maintaining compatibility with platforms on which LMDB does not work?  That was needed by @dtribble some time ago, and I'm not sure if anything's changed since then.

In that case, we need to update this fallback to use a jsonlines-based kvstore and the standard persistent streamStore.

Reply:
heh :)

Yeah, we're looking to get rid of the jsonlines-based kvstore, rather than augment it with the `streamStore` that will hold transcript entries outside the DB. The jsonlines approach could only be used for toy examples anyways (performance-wise), so we weren't *really* useable on platforms without LMDB.

I'll file a ticket for @FUDCo or I to remove it. Such platforms will start to break (or at least work even worse than they currently do) once the next step of this streamStore process lands (moving transcripts from kvStore to streamStore), and will be properly broken when we remove the `check-lmdb.js` fallback.

Reply:
Added in #3131 . @michaelfig if you wanna grab it, feel free (you've got more working knowledge of cosmic-swingset than either of us at this point).

Reply:
> Yeah, we're looking to get rid of the jsonlines-based kvstore

Okay, then my only concern is that the current implementation of the ag-solo's access token needs to keep its state in a portable fashion (i.e., not LMDB which is architecture-dependent).  I leveraged `swing-store-simple` to make that happen.  Its jsonlines implementation will instead need to be inlined in the `agoric-cli` and `solo` packages.

Reply:
Ah, ok I forgot it was also being used for that purpose. We'd need to move that out anyways, since the general plan is to include most of the kvstore contents in the apphash, and a local access token is the very definition of "not part of consensus".

Do you need more than one value? Maybe this could just go into a single one-line file.

Reply:
> Do you need more than one value? Maybe this could just go into a single one-line file.

I can easily implement what I need.  Just please don't remove the jsonlines implementation before I have a chance to do that.  The unit tests for `packages/agoric-cli` don't exercise this requirement (though the `packages/solo` unit tests do).  I'll fix that omission ASAP.

Reply:
gotcha.. I added #3134 to track the removal, with a prohibition on landing it until #3131 is done and cosmic-swingset no longer needs swing-store-simple.